contracts/Content/interfaces/IUniqueContent.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../libraries/LibAsset.sol";
+
+interface IUniqueContent {
+    /*********************** Events *********************/
+    event Mint(address operator, LibAsset.UniqueAssetCreateData data);
+
+    event Burn(address operator, uint256 uniqueId);
+
+    event UniqueUriUpdated(uint256 indexed uniqueId, uint256 indexed version);
+
+    /******** View Functions ********/
+    function originalAssetUri(uint256 _uniqueId, uint256 _version) external view returns(string memory);
+
+    function uri(uint256 _uniqueId, uint256 _version) external view returns(string memory);
```

Comment:
this should be tokenURI(), not uri() because that's the name of the function in ERC721